### PR TITLE
QA doc: import torch before it is used

### PR DESCRIPTION
**Pull request by ByronHsu**
# What does this PR do?

import torch before it is used in qa doc. Otherwise, it complains import error.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@stevhliu

### CODE REVIEW

**Review by stevhliu — APPROVED** [review]
Thanks for the fix! I think you just need to run `make style` to get the tests to pass. Also pinging @amyeroberts for a final review!

File: docs/source/en/tasks/question_answering.mdx [hunk 1]
```diff
@@ -370,7 +370,7 @@ Pass your inputs to the model and return the `logits`:
 
 ```py
 >>> from transformers import AutoModelForQuestionAnswering
```
Comment (amyeroberts) [inline docs/source/en/tasks/question_answering.mdx:372]: nit: sticking with pep8, third party modules should be imported first. 

Annoyingly, github won't let you make suggestions on lines which include deletions, but you'd also need to delete the `import torch` line below

```suggestion
>>> import torch
>>> from transformers import AutoModelForQuestionAnswering
```

**Review by amyeroberts — APPROVED** [review]
@ByronHsu Thanks for you contribution and improving our docs! 

Just a small nit from me. As @stevhliu mentions, you'll need to run `make fixup` locally at the top level of the repo and push any changes to resolve the failing quality checks. Once done, we're good to merge :)

**Comment by ByronHsu** [comment]
@amyeroberts @stevhliu mind checking again? thanks!

**Comment by amyeroberts** [comment]
@ByronHsu Thanks again for fixing! All looks good 👍

**Pull request merged**